8/debian/bullseye/hotspot/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:8u322-b06-jdk-focal as jre-build
+FROM eclipse-temurin:8u332-b09-jre-focal as jre-build
```

Using the JRE rather than the JDK means there is no jar command, which causes test failures.

```suggestion
FROM eclipse-temurin:8u332-b09-jdk-focal as jre-build
```